### PR TITLE
Add style

### PR DESCRIPTION
**Pull request by Metsanpeitto**
:art:  In this branch, I did add a minimum style to the site.

### CODE REVIEW

File: src/components/logic/calculator.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  if (buttonName === '+/-') {
+    if (obj.next) {
+      return { next: (-1 * parseFloat(obj.next)).toString() };
+    }
+    if (obj.total) {
+      return { total: (-1 * parseFloat(obj.total)).toString() };
+    }
+    return {};
+  }
+
```
Comment (DeliceLydia) [inline src/components/logic/calculator.js:89]: - [ ] I highly recommend working on this button `+/-` currently is not working well
s

Got that when I wanted to add `-8` please fix

File: src/components/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    return one.div(two).toString();
+  }
```
Comment (DeliceLydia) [inline src/components/logic/operate.js]: - [ ] When i tried to divide a number by `zero` am getting this error
e

Normally it has to display an `error` on the screen kindly fix that

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
# Hi @Metsanpeitto👏,

Good job so far! your website is almost done, I can navigate on all the 3pages, and the styling is looking good, but to approve it the calculator has to work well, There are some small issues I found you need to fix and you will be ready for the next project but you are almost there.

## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/logic/calculator.js (hunk 1 above)
Comment (Metsanpeitto) [reply src/components/logic/calculator.js:89]: :ok_man: Thanks for pointing it out. I forgot about it. :+1:
I fixed it already :heavy_check_mark:

File: src/components/logic/operate.js (hunk 2 above)
Comment (Metsanpeitto) [reply src/components/logic/operate.js]: :ok:  I didn't realize that possibility. Thanks for noticing.
I got it fixed already :heavy_check_mark:

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**